ParkenDD/LotlistView/LotlistViewController.swift

```diff
@@ -52,6 +52,12 @@ class LotlistViewController: UITableViewController, UIViewControllerPreviewingDe
 		Timer.every(5.minutes, updateData)
 	}
 
+    // works around iOS bug that draws activity spinner in front of tableView
```

Comment:
I like your comment. I'm approving the PR, but I would capitalize the "w" in "works" and add a link to the Trello card so anyone with access to this project and the board can quickly see additional info that may reside in Trello regarding this issue.